db/models/comments.model.js

```diff
@@ -16,19 +16,28 @@ exports.fetchCommentsByArticle = (article_id) => {
                 comments.created_at DESC`, [article_id])
                 .then(({ rows }) => {
                     return rows;
+                })
+                .catch((err) => {
+                    throw err; 
                 });
+        })
+        .catch((err) => {
+            throw err;
```

Comment:
We shouldn't need any catch blocks in the models.
We can promise reject from our models which will then be caught by our controllers and will pass them onto our error handlers.